src/shared/Player/styles.js

```diff
@@ -1,50 +1,147 @@
 import React from 'react';
 import styled from 'styled-components';
```

Comment:
This file was copied from wannameows changes. I haven't had a chance to review it properly but we can do that later.